lib/BiochemistryAPI/BiochemistryAPIImpl.py

```diff
@@ -1,21 +1,21 @@
 # -*- coding: utf-8 -*-
-#BEGIN_HEADER
+# BEGIN_HEADER
```

Comment:
This may be a bad idea ... the `kb-sdk` relies on specific markers in the file to do its validation and auto-formatting. I remember I didn't `black` my `*Impl.py` because it may have messed up `kb-sdk`'s ability to parse the `*Impl.py` file by changing `#BEGIN_HEADER` to `# BEGIN_HEADER`

Reply:
You should be sure about all the other `#BEGIN_HEADER`s and `#END_HEADER`s if you get any problems running `kb-sdk`

Reply:
I went through and fixed those, I'll have to run black with the `--diff` option rather than just let it reformat everything

Reply:
the most recent version of kb-sdk is tolerant to having spaces after the directives, but it depends on what version you're using as to whether it'll be an issue or not